pull_request_template.md

```diff
@@ -1,16 +1,21 @@
-## Purpose
-_Describe the problem or feature in addition to a link to the issues._
+## Type
+- [ ] Feature
+- [ ] Bug
+- [ ] Update (fix something that did not work as expected)
+- [ ] Documentation
 
-## Approach
-_How does this change address the problem?_
+## Purpose
+_Why does this exist?_
 
-#### Open Questions and Pre-Merge TODOs
-- [ ] Use github checklists. When solved, check the box and explain the answer.
+## Solution
+_What changed, and does this address the problem? _
 
-## Learning
-_Describe the research stage_
+## Tests
+_Was this tested? How?_
```

Comment:
I'm not too sure that we need a tests section, just having something on the checklist to make sure that we've remembered to add unit tests.